srcpkgs/zenpower-dkms/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'zenpower-dkms'
+pkgname=zenpower-dkms
+_pkgname=zenpower
+version=0.1.12
+revision=1
+archs="x86_64"
+wrksrc="${_pkgname}-${version}"
+depends="dkms"
+short_desc="Linux kernel driver for reading sensors for AMD Zen family CPUs (DKMS)"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/ocerman/zenpower"
+distfiles="https://github.com/ocerman/$_pkgname/archive/v${version}.tar.gz"
```

Comment:
No need to put `$_pkgname` here, either use the full URL or replace it with `${homepage}`.